docs/src/catalyst_applications/nonlinear_solve.md

```diff
@@ -0,0 +1,99 @@
+# [Finding Steady States using NonlinearSolve.jl](@id nonlinear_solve)
+
+We have previously described how `ReactionSystem` steady states can be found through [homotopy continuation](@ref homotopy_continuation). Catalyst also supports the creation of `NonlinearProblems` corresponding to `ODEProblems` with all left-hand side derivatives set to 0. These can be solved using the variety of nonlinear equation-solving algorithms implemented by [NonlinearSolve.jl](https://github.com/SciML/NonlinearSolve.jl), with the solutions corresponding to system steady states. Generally, this approach is preferred when one only wishes to find *a single solution* (with homotopy continuation being preferred when one wishes to find *all solutions*). 
+
+This tutorial describes how to create `NonlinearProblem`s from Catalyst's `ReactionSystemn`s, and how to solve them using NonlinearSolve. More extensive descriptions of available solvers and options can be found in [NonlinearSolve's documentation](https://docs.sciml.ai/NonlinearSolve/stable/). If you use this in your research, please [cite the NonlinearSolve.jl](@ref nonlinear_solve_citation) and [Catalyst.jl](@ref catalyst_citation) publications.
+
+## Basic example
+Let us consider a simple dimerisation network, where a protein ($P$) can exist in a monomer and a dimer form. The protein is produced at a constant rate from its mRNA, which is also produced at a constant rate.
+```@example nonlinear_solve1
+using Catalyst
+dimer_production = @reaction_network begin
+    pₘ, 0 --> mRNA
+    pₚ, mRNA --> mRNA + P
+    (k₁, k₂), 2P <--> P₂
+    d, (mRNA, P, P₂) --> 0
+end
+```
+This system corresponds to the following ODE:
+```math
+\begin{aligned}
+\frac{dmRNA}{dt} &= pₘ - d \cdot mRNA \\
+\frac{dP}{dt} &= pₚ \cdot mRNA - k₁ \cdot P + 2k₂ \cdot P₂ - d \cdot P \\
+\frac{dP₂}{dt} &= k₁ \cdot P + 2k₂ \cdot P₂ \\
+\end{aligned}
+```
+To find its steady states we need to solve:
+```math
+\begin{aligned}
+0 &= pₘ - d \cdot mRNA \\
+0 &= pₚ \cdot mRNA - k₁ \cdot P + 2k₂ \cdot P₂ - d \cdot P \\
+0 &= k₁ \cdot P + 2k₂ \cdot P₂ \\
+\end{aligned}
+```
```

Comment:
Why not use Latexify to generate these automatically?

Reply:
Latexify has a tendency to jumble up equations a bit, making things come in weird order. Maybe it is better now, will check.